.github/workflows/lint-and-tests.yml

```diff
@@ -28,26 +28,11 @@ permissions:
   # This permission is required by `MishaKav/jest-coverage-comment`
   pull-requests: write
 
-jobs:
-  base:
-    name: Base Tasks
-    runs-on: ubuntu-latest
-    outputs:
-      turbo_args: ${{ steps.turborepo_arguments.outputs.turbo_args }}
-
-    steps:
-      - name: Harden Runner
-        uses: step-security/harden-runner@4d991eb9b905ef189e4c376166672c3f2f230481 # v2.11.0
-        with:
-          egress-policy: audit
```

Comment:
tell me why removing harden runner is a good idea

Reply:
It has not been removed, check the full diff -- both the jobs have their own hardening, this hardening did nothing (other than harden the base job that ran a single shell cmd to export an env var)

Reply:
i see that now, TY TY for the second look

Reply:
Can we make it extra harder? 🤡